dev/io.openliberty.ejbcontainer.v40/src/io/openliberty/ejbcontainer/v40/internal/EJBRuntimeVersionV40.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.ejbcontainer.v40.internal;
```

Comment:
The wiki doesn't have an example for a version specific package name but how about
`io.openliberty.ejb.internal.container.v40`?

Reply:
@tevans78 see my above comments. I consider this a "component" project not directly related to a specific public "ejb" feature.  I view ".ejb." in the package name to be too ambiguous.  Using "ejbcotnainer" makes it easier to identify which aspec of EJB, Managed Beans, and MDB this relates to.... the EJB container, not EJB security, or EJB integration with CDI, JAX-RS, WebContainer, etc.

I prefer to include the full "component" (i.e. project name) prior to the "internal".  Seems to make it easier to avoid multiple project/bundles ever exporting the same package.  Also, with the other ejbcontainer projects, I assume this view would make more sense:

io.openliberty.ejbcontainer.async
io.openliberty.ejbcontainer.async.spi
io.openliberty.ejbcontainer.async.internal
io.openliberty.ejbcontainer.timer
io.openliberty.ejbcontainer.timer.spi
io.openliberty.ejbcontainer.timer.internal
io.openliberty.ejbcontainer.v40
io.openliberty.ejbcontainer.v40.spi
io.openliberty.ejbcontainer.v40.internal

Keep in mind that these are all related to a component, like "transactions", which are included by many features, some with "ejb" in the name, and some not, like "mdb" and "managedBeans".

Reply:
Agreed. I like your examples so I may use those in the wiki. Just need to try and define what a "component" is??